src/dioptra/restapi/v1/experiments/schema.py

```diff
@@ -0,0 +1,85 @@
+# This Software (Dioptra) is being made available as a public service by the
+# National Institute of Standards and Technology (NIST), an Agency of the United
+# States Department of Commerce. This software was developed in part by employees of
+# NIST and in part by NIST contractors. Copyright in portions of this software that
+# were developed by NIST contractors has been licensed or assigned to NIST. Pursuant
+# to Title 17 United States Code Section 105, works of NIST employees are not
+# subject to copyright protection in the United States. However, NIST may hold
+# international copyright in software created by its employees and domestic
+# copyright (or licensing rights) in portions of software that were assigned or
+# licensed to NIST. To the extent that NIST holds copyright in this software, it is
+# being made available under the Creative Commons Attribution 4.0 International
+# license (CC BY 4.0). The disclaimers of the CC BY 4.0 license apply to all parts
+# of the software developed or licensed by NIST.
+#
+# ACCESS THE FULL CC BY 4.0 LICENSE HERE:
+# https://creativecommons.org/licenses/by/4.0/legalcode
+"""The schemas for serializing/deserializing Experiment resources."""
+from __future__ import annotations
+
+from marshmallow import Schema, fields
+
+from dioptra.restapi.v1.groups.schema import GroupRefSchema
+from dioptra.restapi.v1.schemas import (
+    BasePageSchema,
+    GroupIdQueryParametersSchema,
+    PagingQueryParametersSchema,
+    SearchQueryParametersSchema,
+    generate_base_resource_schema,
+)
+
+
+class ExperimentRefSchema(Schema):
```

Comment:
Just to let you know, there's another factory function in https://github.com/usnistgov/dioptra/pull/394 which you can find here `src/dioptra/restapi/v1/schemas.py.` Otherwise the schemas look good to me. I'd recommend starting to implement the mock controller layer so that we can eyeball the swagger docs as well. 

See here for an example of that with Tags: https://github.com/usnistgov/dioptra/blob/v1-tags-schema/src/dioptra/restapi/v1/tags/controller.py

See here for an examples with  Queues:
https://github.com/usnistgov/dioptra/blob/v1-comments_on-schema/src/dioptra/restapi/v1/queues/controller.py
This branch has comments on the schema.py file to help with making the mock controller.